src/components/CartCard/CartCard.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import './CartCard.scss';
+
+const CartCard = ({
+  item,
+  index,
+  handleDecreaseItem,
+  handleIncreaseItem,
+  handleRemoveItem,
+}) => {
+  const { product_name, price, images, quantity } = item;
+
+  return (
+    <div className="cart-card-container">
+      <div className="cart-item">
+        <div className="cart-remove-btn">
+          <button onClick={() => handleRemoveItem(index)}>✕</button>
+        </div>
+        <div className="cart-row-box">
+          <img className="cart-row-img" alt="item" src={images[2]} />
+          <div className="cart-row-expln">
+            <div>
+              <h3 className="expln-title">{product_name}</h3>
+            </div>
+            <div className="cart-row-price">
+              <div className="cart-row-count">
+                <button onClick={() => handleDecreaseItem(index)}>-</button>
+                <span>{quantity}</span>
+                <button onClick={() => handleIncreaseItem(index)}>+</button>
+              </div>
+              <div className="item-price">{`₩${(
+                price * quantity
+              ).toLocaleString()}`}</div>
```

Comment:
큰 차이는 없지만 다음처럼 쓸 수 있을 것 같습니다.
```suggestion
              <div className="item-price">₩{(
                price * quantity
              ).toLocaleString()}</div>
```

Reply:
확인. 감사합니다!